recipes/buildbot-console-view/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "buildbot-console-view" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2a6370db2141aebd7466fee390824fb7190d1e7122d6ebb844690cde141ca2f8"
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - buildbot-pkg
+  run:
+    - python
+    - buildbot-pkg
+    - buildbot-www
+
+test:
+  imports:
+    - buildbot_console_view
+
+about:
+  home: "https://buildbot.net/"
+  license: "GNU General Public (GPL)"
+  license_family: "LGPL"
+  license_file: ""
```

Comment:
Please populate the ``license_file``

Reply:
These packages are not distributing any license files. What to do in that case?

Reply:
You can add the license manually, please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=packaging%20license#packaging-the-license-manually

Reply:
Great, thanks!

Reply:
@marcelotrevisani updated the license file entries.